### PR TITLE
Pixel scale uniformity and FITS keyword documentation

### PR DESCRIPTION
**Pull request by douglase**
addresses #202 by adding a documentation page, increases the number of combinations of keywords checked for in _find_pixelscale_in_headers() and saves a new "pixunit" keyword.

### CODE REVIEW

**Comment by douglase** [comment]
Is anything additional needed to merge this?

File: poppy/poppy_core.py [hunk 1]
```diff
@@ -2738,10 +2740,10 @@ def _find_pixelscale_in_headers(keywords, headers):
             if pixelscale is None and self.planetype is None:
                 # we don't know which keywords might be present yet, so check for both keywords
                 # in both header objects (at least one must be non-None at this point!)
-                _log.debug("  Looking for 'PUPLSCAL' or 'PIXSCALE' in FITS headers to set "
+                _log.debug("  Looking for 'PUPLSCAL' or 'PIXSCALE' or 'PIXELSCL' in FITS headers to set "
                            "pixel scale")
                 keyword, self.pixelscale = _find_pixelscale_in_headers(
-                    ('PUPLSCAL', 'PIXSCALE'),
+                    ('PUPLSCAL', 'PIXSCALE','PIXELSCL'),
```
Comment (mperrin) [inline poppy/poppy_core.py]: I suggest leaving this part of the code unchanged. For back compatibility, we should still allow PIXELSCL in input files we are reading in, even if a different keyword is used in the output files now. We don't want to break the ability to read files which we can currently read ok.

Comment (mperrin) [inline poppy/poppy_core.py]: Wait never mind I read that backwards, you did this right and I confused before and after in the diff. 

Too many hours in a row moving mirrors at Goddard, I'm not thinking straight.

**Comment by mperrin** [comment]
Mostly just getting my attention again long enough to merge is. Thanks @douglase !

File: poppy/poppy_core.py (hunk 1 above)
Comment (douglase) [inline poppy/poppy_core.py]: phew, you had me second guessing

**Pull request merged**